pyatlan/client/credential.py

```diff
@@ -37,17 +37,20 @@ def __init__(self, client: ApiCaller):
         self._client = client
 
     @validate_arguments
-    def creator(self, credential: Credential) -> CredentialResponse:
+    def creator(
+        self, credential: Credential, test: Optional[bool] = True
```

Comment:
I think we can safely remove `Optional` here ^^ since valid values are `True` or `False`, whereas `Optional` implies `None` is also a possibility 🙏

```suggestion
        self, credential: Credential, test: bool = True
```

Reply:
Done